aten/src/ATen/native/native_functions.yaml

```diff
@@ -3332,7 +3332,9 @@
   dispatch:
     CUDA: _fused_rms_norm_cuda
     MPS: _fused_rms_norm_mps
-    CompositeImplicitAutograd: rms_norm_composite
+    CompositeExplicitAutograd: rms_norm_composite
+
+- func: _rms_norm_composite(Tensor input, int[] normalized_shape, Tensor? weight, float? eps) -> (Tensor, Tensor)
```

Comment:
I would argue we don't need this one.
The only reason we kept CompositeImplicitAutograd above is to avoid churn for PT2 world when adding the new version.
But if we are ok with that, then just moving to CompositeExplicitAutograd is enough and we don't need to add this one.

Or do you add this just so you can call it from the decomp?